trashcli/put.py

```diff
@@ -250,71 +254,50 @@ def volume_of_parent(self, file):
 
 
 def get_option_parser(program_name, stdout, stderr):
```

Comment:
Please rename this function to make_parser.

Reply:
Fixed.

Reply:
Should I rename all get_option_parser to make_parser in test file?